dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/validation/ContactValidator.java

```diff
@@ -38,9 +38,9 @@ public void validate(ValidationHelper helper, Context context, String key, Conta
         if (t != null) {
             String url = t.getUrl();
             if (url != null) {
-                if (!ValidatorUtils.isValidURL(url)) {
+                if (!ValidatorUtils.isValidURI(url)) {
                     final String message = Tr.formatMessage(tc, "contactInvalidURL", url);
-                    helper.addValidationEvent(new ValidationEvent(ValidationEvent.Severity.ERROR, context.getLocation("url"), message));
+                    helper.addValidationEvent(new ValidationEvent(ValidationEvent.Severity.ERROR, context.getLocation("uri"), message));
```

Comment:
Shouldn't this be url like here [Contact Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#contactObject)